src/components/Layouts/Primary/NavBar/NavItem/NavItem.tsx

```diff
@@ -101,18 +99,16 @@ export const NavItem: FC<NavItemProps> = ({
     <LeafListItem disableGutters key={title} {...rest}>
       {(href as string).includes('tools') ? (
         <HandoffLink key={title} path={href as string}>
-          <LeafButton style={style}>
+          <MenuItem style={style}>
             {Icon && <Icon style={iconStyle} size="20" />}
             <Title>{title}</Title>
-          </LeafButton>
+          </MenuItem>
         </HandoffLink>
       ) : (
-        <NextLink href={href}>
-          <LeafButton style={style}>
-            {Icon && <Icon style={iconStyle} size="20" />}
-            <Title>{title}</Title>
-          </LeafButton>
-        </NextLink>
+        <MenuItem href={href.toString()} component="a" style={style}>
```

Comment:
Since these aren't `NextLink`s anymore, clicking them causes a full-page navigation and the page goes white for a second. Try it like this and see if this works (I believe it did for me locally):

```tsx
<NextLink href={href} passHref>
  <MenuItem component="a" style={style}>...</MenuItem>
</NextLink>
```

Reply:
Ah yes, the `passHref` attribute.